configure.ac

```diff
@@ -138,9 +143,12 @@ if test "x$enable_android_plugins" = xyes; then
     AC_MSG_ERROR([You need 'xxd' to build the Android plugins])
   fi
   dx_path=`AS_DIRNAME(["$DX"])`
-  ANDROID_CLASSPATH="$dx_path/../../platforms/android-19/android.jar"
+  ANDROID_CLASSPATH="$dx_path/../../platforms/android-20/android.jar"
```

Comment:
I'd vote for making this a for loop to check what the newest available version is, and then remember that in a variable.